internal/ingestion/component/chunker/manual.go

```
@@ -0,0 +1,181 @@
+
+// NewManualChunker constructs the component. It accepts the same heading-level
+// params as TitleChunker (method is pinned to "group"); the position resort is
+// automatic based on whether the upstream payload carries coordinates.
+func NewManualChunker(params map[string]any) (runtime.Component, error) {
+	conf := map[string]any{"method": "group"}
+	for k, v := range params {
+		conf[k] = v
+	}
+	p := defaultsTitle()
+	p.Update(conf)
+	if err := p.TitleChunkerParam.Validate(); err != nil {
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "TitleChunkerParam" from selector

(staticcheck)